Sources/web3swift/KeystoreManager/BIP32Keystore.swift

```diff
@@ -44,8 +41,10 @@ public class BIP32Keystore: AbstractKeystore {
     // --------------
     
     public var keystoreParams: KeystoreParamsBIP32?
-    public var paths: [String:EthereumAddress] = [String:EthereumAddress]()
```

Comment:
My idea to set this variable as deprecated:

`@available(*, deprecated, message: "Please use addressStorage instead")`